trl/trainer/ppo_config.py

```diff
@@ -77,6 +77,10 @@ class PPOConfig(object):
         default=0.2,
         metadata={"help": "Initial KL penalty coefficient (used for adaptive and linear control)"},
     )
+    kl_penalty: Optional[str] = field(
+        default="kl",
+        metadata={"help": "kl penalty. Options: 'kl', 'abs', 'mse'}"},
```

Comment:
can we add a little bit more description here (same for the imdb args)?

Reply:
@lvwerra I updated the description, let me know if you require any further improvements.